src/stk/molecular/topology_graphs/metal_complex/metal_complex.py

```diff
@@ -0,0 +1,311 @@
+"""
+Metal Complex
+=============
+
+.. toctree::
+    :maxdepth: 2
+
+    Paddlewheel <\
+    stk.molecular.topology_graphs.metal_complex.paddlewheel.paddlewheel\
+>
+    Porphyrin <\
+stk.molecular.topology_graphs.metal_complex.porphyrin.porphyrin\
+>
+    Octahedral <\
+stk.molecular.topology_graphs.metal_complex.octahedral.octahedral\
+>
+    Octahedral Lambda <\
+stk.molecular.topology_graphs.metal_complex.octahedral.octahedral_lambda\
+>
+    Octahedral Delta <\
+stk.molecular.topology_graphs.metal_complex.octahedral.octahedral_delta\
+>
+    Square Planar <\
+stk.molecular.topology_graphs.metal_complex.square_planar.square_planar\
+>
+    Bidentate Square Planar <\
+stk.molecular.topology_graphs.metal_complex.square_planar.bidentate_square_planar\
+>
+    Cis Protected Square Planar <\
+stk.molecular.topology_graphs.metal_complex.square_planar.cis_protected_square_planar\
+>
+
+"""
+
+from ..topology_graph import TopologyGraph
+from ...reactions import GenericReactionFactory
+
+
+class MetalComplex(TopologyGraph):
+    """
+    Represents a metal complex topology graph.
+
+    Notes
+    -----
+    *Subclass Implementation*
+
+    Each subclass needs to define the iterable attributes,
```

Comment:
I don't think any iterable would work here, for example:
`_metal_vertex_prototypes = (vertex for vertex in vertices)`
where `vertices` is some iterable of `Vertex` instances.

Reply:
Done